Feat: Fetcher Plugin System (#277)
SE-UUlm/snowballr-backend: wiki/Configuration.md

| `SMTP_SENDER_NAME`                     |          :white_check_mark:          |       -       | Name of the sender for emails.                                                                         |
| `SMTP_SENDER_EMAIL`                    |          :white_check_mark:          |       -       | Email address of the sender for emails.                                                                |
| `SENSITIVE_INFORMATION_RETENTION_DAYS` |                 :x:                  |      30       | Number of days sensitive information will be kept after an entity got deleted.                         |
| `WEB_PORT`                             |                 :x:*                 |     8081      | The port where the proxy is served (used for gRPC-Web).                                                |
| `BACKEND_TAG`                          |                 :x:*                 | `latest-dev`  | Tag of registry backend image to use for `registry` docker compose profile.                            |

> | `PLUGIN_DIRECTORY`                     |                 :x:                  | `./plugins`   | The directory in which the plugins reside. Defaults to `plugins` inside the current working directory. |

Reviewer: ```suggestion
| `PLUGIN_DIRECTORY`                     |                 :x:                  | `./plugins`   | The directory in which the plugins reside. The directory is relative to the root directory of the project. |
```
